docs/reference/api/widgets/slider.rst

```diff
@@ -9,10 +15,19 @@ Slider
    :exclude: {0: '(?!^(Slider|Component)$)'}
 
 
-
 Usage
 -----
 
+.. code-block:: Python
+
+    import toga
+
+    def my_callback(slider):
+        print(slider.value)
+
+    slider = toga.Slider(value=5, range=(-10, 10), on_change=my_callback)
+
+
```

Comment:
I think we need some overall docs describing the various modes of operation - continuous vs discrete etc. They don't need to go into detail - the docs for `tick_marks` etc detail the specifics; but at a high level, there should be a description of the broad capabilities of the widget, and some pointers into the attributes of interest that control those capabilities.

Reply:
Done.